lib/Embargo/Embargo.js

```diff
@@ -5,6 +5,7 @@ import { Layout } from '@folio/stripes/components';
 
 export default class Embargo extends React.Component {
     static propTypes = {
+      alignment: PropTypes.string,
```

Comment:
Following the comment [here](https://github.com/folio-org/ui-agreements/pull/629#pullrequestreview-503477149) you could change this proptype to be 
```suggestion
      alignment: PropTypes.oneOf(['left', 'right', 'center'])
```